### ISSUE TITLE
Disable SSL does not handled correct

### ISSUE THREAD

**Issue by nikita-gorodeckij** [BitGo/BitGoJS#1079]
## Expected Behavior
I play with BitGo api  and found that flag `BITGO_DISABLE_SSL` is not working

I run:

```
docker run \
        -e BITGO_ENV="prod" \
        -e BITGO_DISABLE_SSL="true" \
        -it -p 4445:4445 bitgosdk/express:latest  --port 4445
```

Also I tried

```
docker run \
        -e BITGO_ENV="prod" \
        -e BITGO_DISABLE_SSL="1" \
        -it -p 4445:4445 bitgosdk/express:latest  --port 4445
```

But I always catch error: `Error: Must enable TLS when running against prod and listening on external interfaces!`

However, this code works fine:

```

docker run \
        -e BITGO_ENV="prod" \
        -it -p 4445:4445 bitgosdk/express:latest  --port 4445 --disablessl
```

Is it my mistake or it is docker image's bug?

**Comment by tylerlevine** [comment]
Thanks for opening the issue, I'll look into this.

**Comment by rakveli** [comment]
Hello my exchange wallets are showing un able to connect to blockchain in the production mode but everything is working fine on the test mode. please kindly look into this ?

**Comment by tylerlevine** [comment]
This should be fixed by #1094, could you confirm that this issue has been resolved for you with the latest bitgo express image @nikita-gorodeckij?

**Comment by tylerlevine** [comment]
> Hello my exchange wallets are showing un able to connect to blockchain in the production mode but everything is working fine on the test mode. please kindly look into this ?

@rakveli, could you please open a separate issue for this and provide some more details?
